MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m

```diff
@@ -128,8 +119,58 @@ + (void)openUrl:(NSURL *)url {
   [[self sharedInstance] openUrl:url];
 }
 
++ (void)notifyUserUpdateAction:(MSUserUpdateAction)action {
+  [[self sharedInstance] notifyUserUpdateAction:action];
+}
+
++ (void)setDelegate:(id<MSDistributeDelegate>)delegate {
+  [[self sharedInstance] setDelegate:delegate];
+}
+
 #pragma mark - Private
 
+- (void)startUpdate {
+  if ([self isEnabled]) {
```

Comment:
Well, that cannot be if it is called from another place. BTW, I found this has a bug and some of changes made in here will be introduced in Update Dialog change PR. I want to close it temporary and then reopen once it is ready for merge.